src/batcher/input_parser.star

```diff
@@ -5,6 +5,8 @@ _registry = import_module("/src/package_io/registry.star")
 _DEFAULT_ARGS = {
     "image": None,
     "extra_params": [],
+    "pprof_enabled": False,
+    "pprof_port": _net.port(number=6060),
```

Comment:
is pprof_port actually used? I think only the helper is defining the port now, no?
(also, I don't think we'd necessarily want to customize the internal port anyway)

Reply:
[99e7117](https://github.com/ethpandaops/optimism-package/pull/352/commits/99e711704d536fd7fa6ebced856991084069b13a)